bot/internal/bot/remind.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bot
+
+import (
+	"context"
+	"strings"
+
+	"github.com/gravitational/shared-workflows/bot/internal/github"
+	"github.com/gravitational/trace"
+)
+
+const newDocsReminderList = `Thanks for writing a new docs page!
+
+Please don't forget to:
+
+- [ ] Read the [Docs Contribution Guide](https://goteleport.com/docs/contributing/documentation/) if you haven't already.
+- [ ] Add backport labels (&#96;backport/branch/v<number>&#96;) for the current and two major Teleport versions (if applicable for your feature) 
+- [ ] Make sure you list any new docs pages in &#96;docs/config.json&#96;.
+- [ ] Document new functionality in a how-to guide, reference docs, and architectural docs (or create new issues to do this later).
```

Comment:
I am still learning about the process we have for generating our documentation.  How does the team learn about the details of a feature in order to document it?

Does it make sense to add an item to review the RFD in detail?  In a couple cases we are adding in details for when the feature docs are being crafted, for example: https://github.com/gravitational/teleport.e/pull/911/commits/c4838882fab48acedcbf611c3e0095a185d69253

Reply:
> How does the team learn about the details of a feature in order to document it?

The person writing the first draft of the docs is almost always the dev who coded the feature, so this is not necessary.